### ISSUE TITLE
Can't unfollow more than 10 people

### ISSUE THREAD

**Issue by alexauvray** [InstaPy/InstaPy#745]
The bot goes to sleep after 10 people like it's supposed to. But then it stops. Any idea ?

```
sleeping for about 10min
--> Total people unfollowed : 10 

Session ended
-------------
```

**Comment by converge** [comment]
@alexsonnay are you using this method ?
session.unfollow_users(amount=10, onlyInstapyFollowed = True, onlyInstapyMethod = 'FIFO' )
amount is the number of profile you will unfollow

**Comment by alexauvray** [comment]
Yes.
I tried with `session.unfollow_users(amount=50)`

**Comment by converge** [comment]
@alexsonnay how is all your login/actions/options ?

**Comment by alexauvray** [comment]
Everything is fine on this matter :). It just ends the session quicker than it's supposed to
```
[2017-10-15 12:21:47] --> Total people unfollowed : 10
[2017-10-15 12:21:47] 
[2017-10-15 12:21:47] Session ended
```
I just have a session.set_dont_include and a session.unfollow_users in the options.

**Comment by jasetom** [comment]
I have the same problems except the script unfollows 19 people and then stops?

**Comment by converge** [comment]
@alexsonnay @jasetom it will be helpful if you can provide detailed information about it, so we can reproduce it.

- what exatcly setup/options/actions are you using and how is it setup ?
- what is the stacktrace/error log

**Comment by jasetom** [comment]
I'm only using the unfollow method. My whole script contains only these lines:
```
from instapy import InstaPy

insta_username = 'username'
insta_password = 'pass'

session = InstaPy(username=insta_username, password=insta_password)
session.login()
session.unfollow_users(amount=1000)
session.end()
```

Terminal prints out this: (I've changed the names of ig accounts to x for privacy)
```
Logged in successfully!
('set_automated_followed_pool error \n', "[Errno 2] No such file or directory: './logs/username_followedPool.csv'")
--> Ongoing Unfollow 1, now unfollowing: x
--> Ongoing Unfollow 2, now unfollowing: x
--> Ongoing Unfollow 3, now unfollowing: x
--> Ongoing Unfollow 4, now unfollowing: x
--> Ongoing Unfollow 5, now unfollowing: x
--> Ongoing Unfollow 6, now unfollowing: x
--> Ongoing Unfollow 7, now unfollowing: x
--> Ongoing Unfollow 8, now unfollowing: x
--> Ongoing Unfollow 9, now unfollowing: x
--> Ongoing Unfollow 10, now unfollowing: x
sleeping for about 10min
--> Ongoing Unfollow 11, now unfollowing: x
--> Ongoing Unfollow 12, now unfollowing: x
--> Ongoing Unfollow 13, now unfollowing: x
--> Ongoing Unfollow 14, now unfollowing: x
--> Ongoing Unfollow 15, now unfollowing: x
--> Ongoing Unfollow 16, now unfollowing: x
--> Ongoing Unfollow 17, now unfollowing: x
--> Ongoing Unfollow 18, now unfollowing: x
--> Ongoing Unfollow 19, now unfollowing: x
--> Total people unfollowed : 19

Session ended
-------------
```
Other information:
pip 9.0.1
python 2.7
ChromeDriver 2.33
MacOS Sierra 10.12.6

**Comment by alexauvray** [comment]
Yep same for me now it's 19. I have found some kind of a solution : I'm using a 'while true' with a 'sleep' function in the initial command in the terminal to repeat the python script every 10 minutes ;)

**Comment by converge** [comment]
can someone test the bug fix before merging ? #796

**Comment by jasecoop** [comment]
I'm getting the same problem here...

> ("--> Total unfollowNum reached it's amount given ", 10)
> --> Total people unfollowed : 10
>
> Session ended
> -------------

Any idea of the issue?

**Comment by converge** [comment]
@jasecoop I wrote a bug fix #796 for it some days ago, is your repository updated ?
